css/src/components/form/checkbox.scss

```diff
@@ -116,4 +140,40 @@ $checkbox-spacing: $spacer-3 !default;
 			border-color: $checkbox-check-color;
 		}
 	}
+
+	&.is-invalid input[type='checkbox']:not(:checked) {
+		+ .checkbox-check {
+			border-color: $checkbox-danger-border-color;
+		}
+
+		@include focus-visible() {
+			+ .checkbox-check {
+				box-shadow: $checkbox-focus-box-shadow-size $checkbox-danger-border-color;
+			}
+		}
+	}
+
+	&.is-valid input[type='checkbox']:not(:checked) {
+		+ .checkbox-check {
+			border-color: $checkbox-success-border-color;
+		}
+
+		@include focus-visible() {
+			+ .checkbox-check {
+				box-shadow: $checkbox-focus-box-shadow-size $checkbox-success-border-color;
+			}
+		}
+	}
+
+	&:hover {
+		&:not(.is-valid):not(.is-invalid) {
```

Comment:
Apologies for only just thinking of this, but what is the use case for `is-valid`? Could we remove it? Wouldn't the default state imply valid?

Reply:
`is-valid` was removed